planner/core/plan_cost.go

```diff
@@ -509,14 +477,7 @@ func (p *PhysicalIndexScan) GetPlanCost(_ property.TaskType, option *PlanCostOpt
 		rowSize = p.getScanRowSize()
 		selfCost = rowCount * rowSize * scanFactor
 	case modelVer2:
-		scanFactor = p.ctx.GetSessionVars().GetScanFactor(p.Table)
-		if p.Desc {
-			scanFactor = p.ctx.GetSessionVars().GetDescScanFactor(p.Table)
-		}
-		rowCount = getCardinality(p, costFlag)
-		rowSize = math.Max(p.getScanRowSize(), 2.0)
-		logRowSize := math.Log2(rowSize)
-		selfCost = rowCount * logRowSize * scanFactor
+		return p.getPlanCostVer2(taskType, option)
 	}
 	if option.tracer != nil {
 		setPhysicalTableOrIndexScanCostDetail(p, option.tracer, rowCount, rowSize, scanFactor, costModelVersion)
```

Comment:
It seems we not use the trace for the version2.

Reply:
Yes. I'll support it for ver2 later in the next a few PRs.